costmap_2d/include/costmap_2d/inflation_layer.h

```diff
@@ -103,16 +103,17 @@ class InflationLayer : public Layer
    * @return A cost value for the distance */
   virtual inline unsigned char computeCost(double distance) const
   {
+    double inscribed_radius = inflate_inscribed_ ? inscribed_radius_ : 0.0;
```

Comment:
S: This is extremely error prone, since the only different between your variable and the class attr is a single `_`.
I would suggest renaming to something else.

Reply:
well, that's because they represent the same physical property (but under different circumstances)